modules/dxp/apps/saml/saml-opensaml-integration/src/test/java/com/liferay/saml/opensaml/integration/internal/servlet/profile/WebSsoProfileIntegrationTest.java

```diff
@@ -512,14 +495,6 @@ public void testDecodeAuthnRequestStageAuthenticated() throws Exception {
 		Assert.assertEquals(
 			inboundSamlMessageId, messageInfoContext.getMessageId());
 
```

Comment:
same here

Reply:
Hey @jkappler, when I remove this line, `ant format-source-local-changes` gives me the following message:
```
There should be an empty line before line "496", as we finished referencing variable "messageInfoContext", see https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/documentation/check/missing_empty_line_check.markdown: ./modules/dxp/apps/saml/saml-opensaml-integration/src/test/java/com/liferay/saml/opensaml/integration/internal/servlet/profile/WebSsoProfileIntegrationTest.java 496 (Checkstyle:MissingEmptyLineCheck)
```

I left this line in because of that message.